sparkmagic/sparkmagic/handlers.py

```diff
@@ -0,0 +1,95 @@
+import json
+from notebook.utils import url_path_join
+from notebook.base.handlers import IPythonHandler
+
+from sparkmagic.kernels.kernelmagics import KernelMagics
+from sparkmagic.utils.sparkevents import SparkEvents
+
+
+class ReconnectHandler(IPythonHandler):
+    def post(self):
```

Comment:
I looked at some other IPythonHandler.
They annotate HTTP methods with "@web.authenticated".
I know we don't use HTTP authentication in our setup. But it might be a good idea to follow the practice.